README.md

```diff
@@ -78,6 +88,17 @@ The Simplify jar will be in `simplify/build/libs/simplify.jar`. You can test it'
 java -jar simplify/build/libs/simplify.jar -it 'org/cf' simplify/obfuscated-example
 ```
 
+**Windows users**
+
+Using CMD:
+
+```
+cd [location of your simplify folder]
+gradlew fatjar
```

Comment:
The main windows / linux difference is in this command. I think it's a lot cleaner to keep this section mostly as it is, and split up this command into linux / windows sections.

Reply:
Hum... I don't understand what this means, sorry... Should I move it? Where?

Reply:
Create the Windows / Linux distinction above, for the build command. That's the only real difference in the build instructions.

Reply:
Alright.